platforms-serverless/azure-functions-linux/func-placeholder/index.ts

```diff
@@ -1,7 +1,13 @@
-import { PrismaClient, Prisma } from '@prisma/client'
-import { AzureFunction, Context, HttpRequest } from '@azure/functions'
+import { Context, HttpRequest } from '@azure/functions'
+import { Prisma, PrismaClient } from '@prisma/client'
 
-const client = new PrismaClient()
+const client = new PrismaClient({
+  datasources: {
+    db: {
+      url: process.env.AZURE_FUNCTIONS_LINUX_PG_URL,
+    },
+  },
+})
```

Comment:
We should technically only do this for Node-API.

Reply:
Removed